### DISCUSSION TITLE
PMDG DC-6 gauges

### DISCUSSION THREAD

**Discussion by Tbonekiller** [nguyenquyhy/Flight-Tracker-StreamDeck#278] [Q&A]
Hi. 

First of all, thank you for a greate plugin.

I would like to use Flight Tracker with PMDG DC-6. Problem is that values displayed on my stream deck does not match with the gauge i the simulator.

ENG_OIL_TEMPERATURE__1 does not show the same value as in the simulator. 

Same with ENG_CYLINDER_HEAD_TEMPERATURE__1.

Is it possible to "offset"?

Thx Thomas

**Comment by nguyenquyhy** [comment]
Have you tried to put in a unit?

**Reply by Tbonekiller** [reply]
No, Is there an explanation for the unit option, I´m not sure what I´m suppose to type there.

**Reply by nguyenquyhy** [reply]
Those in here https://docs.flightsimulator.com/html/index.htm#t=Programming_Tools%2FSimVars%2FSimulation_Variable_Units.htm .

**Reply by Tbonekiller** [reply]
Thanks. I´ve just tried it. It still dosen´t match, it is about 30c off. Maybe it is a pmdg "thing"
